### PR TITLE
[WIP] Homebrew formula

### PR DESCRIPTION
**Pull request by etaque**

### CODE REVIEW

**Comment by etaque** [comment]
[Homebrew](https://trello.com/c/JVN9zTzT/133-homebrew)

**Comment by etaque** [comment]
[Homebrew](https://trello.com/c/JVN9zTzT/133-homebrew)

File: Formula/dividat-driver.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# setup HOMEBREW_GITHUB_API_TOKEN from https://github.com/settings/tokens with *repo* scope (Full control of private repositories)
```
Comment (knuton) [inline Formula/dividat-driver.rb]: I think this and the use of `GitHubPrivateRepositoryDownloadStrategy` are no longer needed?

Comment (etaque) [reply Formula/dividat-driver.rb]: good catch!

**Pull request closed without merging**